components/UserSavedCard.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div>
+    <div v-if="isDefaultCard()" class="is-primary">Primary</div>
+    <div class="card-list">
+      <div class="card-list__left">
+        <div>Icon</div>
+      </div>
+      <div class="card-list__right">
+        <div>Ending :{{ paymentMethod.endingDigit }}</div>
+        <div>Exp : {{ paymentMethod.expiry }}</div>
+      </div>
+      <div v-if="showActions" class="card-list__actions">
+        <div class="card-list__edit" @click="$emit('click:edit-card', paymentMethod)">Edit</div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { defineComponent } from "@vue/composition-api"
+export default defineComponent({
+  name: "UserSavedCard",
+  props: {
+    paymentMethod: {
+      type: Object,
+      default: () => {},
+    },
+    showActions: {
+      type: Boolean,
+      default: false,
+    },
+  },
+
+  setup() {
+    const isDefaultCard = () => {
```

Comment:
make it computed

Reply:
I think not required as it will render once.